submit-api/src/submit_api/services/invitation_service.py

```diff
@@ -206,6 +206,14 @@ def get_valid_invitation(token):
         if invitation.expiry_date < datetime.datetime.utcnow():
             return {"error": "Invitation has expired"}, False
 
+        # Retrieve account from account_id
+        account = AccountModel.query.filter_by(id=invitation.account_id).first()
```

Comment:
Isn't the account already joined to the invitation
![image](https://github.com/user-attachments/assets/6fda336a-d8aa-4265-9d27-598d00e96613)

Reply:
can't we in the schema use fields.Pluck on the account to get the proponent_id? or a even a method to just access it and return it

Reply:
i used fields.pluck and it was not returning the proponent_id